docs/solr-cloud/solr-cloud-crd.md

```diff
@@ -1057,7 +1063,7 @@ A few aspects of the default `security.json` configuration warrant a closer look
         "name": "k8s-probe-0",
         "role": null,
         "collection": null,
-        "path": "/admin/info/health"
+        "path": "/admin/info/system"
```

Comment:
Just to match the change I made [here](https://github.com/apache/solr-operator/pull/680/files#diff-0523160cf0d0ae7e25c5418c19afdb147ac11980981e5cfc6051870653b5ac51R1003). So that the `k8s-probe-0` object in both places has the same `path`.

Reply:
I C